Sources/Views/TODO.swift

```diff
@@ -12,6 +12,12 @@ extension HtmlUpgrade.Attribute {
   }
 }
 
+extension HtmlUpgrade.Attribute {
+  public static func playsinline<T>(_ value: Bool) -> HtmlUpgrade.Attribute<T> {
+    return .init("playslinline", value ? "" : nil)
```

Comment:
Ah dang do we need to add this to `swift-html`?

Reply:
yeah seems like it